features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -348,6 +348,46 @@ Feature: Gift Exchange Challenge
     Then I should see "myname1"
       And I should see "Fulfilled Story"
 
+  Scenario: A mod can default all incomplete assignments
```

Comment:
It looks like the indenting on this went a little wonky -- the `Given` and `And`s here are right, but everything else is wrong. Specifically, each line starting with `When` or `Then` is indented 3 spaces and each line starting `And` is 5.